data/locale/en-US.ini

```diff
@@ -46,6 +46,8 @@ KeyFrames="Key Frames"
 KeyFrames.IntervalType="Interval Type"
 KeyFrames.IntervalType.Frames="Frames"
 KeyFrames.IntervalType.Seconds="Seconds"
+KeyFrames.IntervalType.Description="Keyframe interval type"
+KeyFrames.Interval.Description="Distance between key frames, in frames or seconds.\nHigher values allow for more efficient compression,\nat the expense of slower seeking; some video editors may\nsuffer a performance decrease. Default is 2 seconds."
```

Comment:
Description text is incorrect, higher values are not correlated with more efficient compression. Higher values may cause less efficient compression due to no proper reference frame being left. Information about video editors should also not be added here as it is specific to certain video editors and not a general observation that applies to all of them.

Reply:
For example, Resolve works perfectly fine with keyframes every 15 seconds, but Adobe Premiere Pro struggles with it.